terraform/github_members.tf

```diff
@@ -99,6 +99,12 @@ resource "github_team_membership" "doc-writers_nilium" {
 
 # Maintainers
 
+resource "github_team_membership" "pkg-committers_chocimier" {
+  team_id = "${github_team.pkg-committers.id}"
+  role = "maintainer"
```

Comment:
This should be member, not maintainer.  It should also appear in the section below, not this one.